### PR TITLE
Added note for Obsolete methods

### PR DESCRIPTION
**Pull request by eshanrnh**
## Description

_What did you add/update/change?_

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

CMS v15

## Deadline (if relevant)

_When should the content be published?_

### CODE REVIEW

File: 15/umbraco-cms/.gitbook/includes/obsolete-warning-snapshot.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: 'Obsolete Warning: Snapshot'
+---
+
+{% hint style="warning" %}
+The following example uses `using Umbraco.Cms.Core.PublishedCache`, `IPublishedSnapshotAccessor`, and `PropertyCacheLevel.Snapshot`, which are obsolete in Umbraco 15 and will be removed in a future version. For more information, see the [Version specific upgrades](../../fundamentals/setup/upgrading/version-specific/#umbraco-15) article.
```
Comment (sofietoft) [inline 15/umbraco-cms/.gitbook/includes/obsolete-warning-snapshot.md:6]:
```suggestion
The following example uses `Umbraco.Cms.Core.PublishedCache`, `IPublishedSnapshotAccessor`, and `PropertyCacheLevel.Snapshot`, which are obsolete in Umbraco 15 and will be removed in a future version. For more information, see the [Version specific upgrades](../../fundamentals/setup/upgrading/version-specific/#umbraco-15) article.
```

File: 15/umbraco-cms/reference/webhooks/expanding-webhook-events.md [hunk 2]
```diff
@@ -140,6 +140,8 @@ To leverage the `WebhookEventContentBase<TNotification, TEntity>` class, follow
 
    If we take a look at the `ContentPublishedWebhookEvent`, we can see how these methods are overriden.
 
+    {% include "../../../.gitbook/includes/obsolete-warning-ipublishedsnapshotaccessor.md" %}
```
Comment (sofietoft) [inline 15/umbraco-cms/reference/webhooks/expanding-webhook-events.md:143]:
```suggestion
{% include "../../../.gitbook/includes/obsolete-warning-ipublishedsnapshotaccessor.md" %}
```

**Review by sofietoft — COMMENTED** [review]
Looks good. Just a couple of comments.

**Comment by eshanrnh** [comment]
@sofietoft this PR is now ready to be verified 😀

**Review by sofietoft — APPROVED** [review]
I've done some randoms checks in the preview, and it looks like everything checks out 💪

**Pull request merged**